docs/paper.md

```diff
@@ -42,28 +42,33 @@ The research purpose of this software is (a) to mathematically verify and valida
-- includes novel scores not commonly found elsewhere (e.g. FIRM, Flip-Flop Index).
-- is designed to work effectively with the libraries, data structures and methods commonly used in the meteorology, weather and climate communities. Scores can effectively handle the dimensionality, data size and data structures commonly utilised for: 
-  - gridded earth system data (e.g. Numerical Weather Prediction models) 
-  - tabular, point, lat/lon or site-based data (e.g. forecasts for specific locations)
-- includes a companion Jupyter Notebook for each score, metric and statistical test to demonstrate its use in practice
+- is designed to work with n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly utilised for: 
+  - gridded earth system data (e.g. Numerical Weather Prediction models)
+  - tabular, point, lat/lon or site-based data (e.g. forecasts for specific locations).
+- it is designed to handle missing data, masking of data and weighting of results.
+- includes a companion Jupyter Notebook for each score, metric and statistical test to demonstrate its use in practice.
+- includes novel scores not commonly found elsewhere (e.g. FIRM [@Taggart:2022a], Flip-Flop Index [@Griffiths:2019; @griffiths2021circular]).
 - is highly modular and avoids extensive dependencies by providing its own implementations where relevant.
 - is intended to be easy to integrate and utilise in a wide variety of environments. It has been tested and used on workstations, servers and in high performance computing (supercomputing) environments. 
-- utilises Dask for scaling and performance
+- utilises Dask for scaling and performance.
 
-## Roadmap and Future Development
+## Metrics, Statistical Techniques and Data Processing Tools Included in Scores 
 
-At the time of writing, the scores contained in this package are: Mean Squared Error (MSE), Mean Absolute Error (MAE), Root Mean Squared Error (RMSE), the Fixed Risk Multicategorical (FIRM) score [@Taggart:2022a], Continuous Ranked Probability Score (CRPS) for Cumulative Distribution Functions (CDFs) (including threshold-weighting, see [@Gneiting:2011]), CRPS for ensembles [@Gneiting_2007; @Ferro_2013], the Flip-Flop Index [@Griffiths:2019; @griffiths2021circular], Receiver Operating Characteristic (ROC) curves, the quantile score, and the Murphy score [@Ehm:2016]. It also includes the Diebold-Mariano statistical test [@Diebold:1995] with both the [@Harvey:1997] and [@Hering:2011] modifications. Additionally it contains isotonic regression which is becoming an increasingly important tool in forecast verification and can be used to generate stable reliability diagrams [@dimitriadis2021stable]. We anticipate more scores, metrics and statistical techniques will be added over time.
```

Comment:
Note: all of the content in the above paragraph is still in the proposed alternative draft: 
- All of the scores above are in the new table. 
- As I deleted this paragraph, I moved the references into the table instead.
- I relocated the sentence about isotonic regression into the "Summary" section
- I relocated the sentence about more metrics etc. being added over time